Error installed latest version ` polkadot-api@1.15.3`

Hey 👋 ,  I'm using `papi` in one of our projects and the latest version is failing with an error at install process.

To reproduce, you can use this simple package.json file and try to install the deps (with both npm or pnpm).

// package.json
```json
{
  "name": "papi-issue",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "polkadot-api": "^1.14.1"
  }
}
```

Install npm

```bash
❯ npm i
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*
```

Install pnpm

```bash
❯ pnpm i
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@polkadot-api/json-rpc-provider@workspace:*" is in the dependencies but no package named "@polkadot-api/json-rpc-provider" is present in the workspace

This error happened while installing the dependencies of polkadot-api@1.15.3
 at @polkadot-api/substrate-client@0.4.2
 at @polkadot-api/raw-client@0.1.0

Packages found in the workspace:
Progress: resolved 24, reused 24, downloaded 0, added 0
```

Thanks!!

--- a/packages/cli/CHANGELOG.md
+++ b/packages/cli/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 0.14.6 - 2025-08-08
+
+### Fixed
+
+- Update dependencies
+
 ## 0.14.5 - 2025-08-07
 
 ### Fixed

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@polkadot-api/cli",
-  "version": "0.14.5",
+  "version": "0.14.6",
   "author": "Victor Oliva (https://github.com/voliva)",
   "license": "MIT",
   "sideEffects": true,

--- a/packages/client/CHANGELOG.md
+++ b/packages/client/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 1.15.4 - 2025-08-08
+
+### Fixed
+
+- Fixed broken dependency link
+
 ## 1.15.3 - 2025-08-07
 
 ### Fixed

--- a/packages/client/package.json
+++ b/packages/client/package.json
@@ -1,6 +1,6 @@
 {
   "name": "polkadot-api",
-  "version": "1.15.3",
+  "version": "1.15.4",
   "author": "Josep M Sobrepere (https://github.com/josepot)",
   "repository": {
     "type": "git",

--- a/packages/docgen/CHANGELOG.md
+++ b/packages/docgen/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 0.2.14 - 2025-08-08
+
+### Fixed
+
+- Update dependencies
+
 ## 0.2.13 - 2025-08-07
 
 ### Fixed

--- a/packages/docgen/package.json
+++ b/packages/docgen/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@polkadot-api/docgen",
-  "version": "0.2.13",
+  "version": "0.2.14",
   "author": "Yuri Volkov (https://github.com/mutantcornholio)",
   "license": "MIT",
   "sideEffects": true,

--- a/packages/raw-client/CHANGELOG.md
+++ b/packages/raw-client/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 0.1.1 - 2025-08-08
+
+### Fixed
+
+- Fixed broken dependency link
+
 ## 0.1.0 - 2025-08-08
 
 ### Changed

--- a/packages/raw-client/package.json
+++ b/packages/raw-client/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@polkadot-api/raw-client",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "author": "Josep M Sobrepere (https://github.com/josepot)",
   "repository": {
     "type": "git",

--- a/packages/substrate-client/CHANGELOG.md
+++ b/packages/substrate-client/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 0.4.3 - 2025-08-08
+
+### Fixed
+
+- Update dependencies
+
 ## 0.4.2 - 2025-08-07
 
 ### Fixed

--- a/packages/substrate-client/package.json
+++ b/packages/substrate-client/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@polkadot-api/substrate-client",
-  "version": "0.4.2",
+  "version": "0.4.3",
   "author": "Josep M Sobrepere (https://github.com/josepot)",
   "repository": {
     "type": "git",

